src/models/Project.ts

```diff
@@ -48,8 +48,7 @@ import i18n from "../locales";
 export interface ProjectData {
     name: string;
     description: string;
-    code: string;
-    awardNumber: string;
+    awardNumber: number;
```

Comment:
Finally, I've implemented it as a string (JS does not has integer/float types, which is annoying), but I've kept the number validator for the future.